### DISCUSSION TITLE
Translating to a new language

### DISCUSSION THREAD

**Discussion by saevarom** [wagtail/guide#65] [General]
I am interested in translating the guide to my native language, Icelandic.

I get the impression that the methods for translating content is still under development, but I'm interested in what the plans are for translating content going forward?

Will you have translation teams for each language? Something like the translation teams in Transifex?

**Comment by lb-** [comment]
Raised this as an issue https://github.com/wagtail/guide/issues/67

**Comment by thibaudcolas** [comment]
@saevarom would love to see this happen!

A lot of the site is still under development, not just translations. What I can say on translations in particular is that:

- We want to use wagtail-localize (#62)
- We’ve not had any discussions (that I’m aware of) on how we manage translations over time

**Comment by thibaudcolas** [comment]
For anyone interested in this – we now have a brief mention of our interest in more translations on https://guide.wagtail.org/en-latest/contributing/#translations. Best place to get involved is to join the editor-guide channel on the Wagtail Slack.

**Comment by f4biosa** [comment]
And I am interested in translating the guide to my native language, Brazilian portuguese.

**Reply by lb-** [reply]
@f4biosa awesome - to get involved with translations you can follow the steps here https://guide.wagtail.org/en-latest/contributing/#translations

Basically, reach out on the channel in Slack and we can get you a username to edit the translations.
